### PR TITLE
GH-1306 ACC Pins

### PR DESCRIPTION
**Pull request by mlzummo**
Adds Pablo's SVG pins to src/map/Pins.js
Fixes GH-1306

Signed-off-by: jay <jbrundage372@gmail.com>
Signed-off-by: jay <mlzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith the code for doing auto text wrapo is what was in text.js you mentioned in the other PR ... however if we want to leave it so that wrap only happens on \n and not based on width let me know ... i messed with it a bit ... but i couldnt get the width calcs correct so it was being a pain... so let me know

@jbrundage ... thoughts

File: src/map/Pins.js [hunk 1]
```diff
@@ -29,47 +33,143 @@
...
-                var pos = base.project(d[0], d[1]);
-                if (!pos) {
-                    return "none";
+        this.pinsPaths.enter().append("g").each(function(d) {
+
+            d3.select(this).style("opacity", d[2] && d[2].opacity ? d[2].opacity : context.opacity()); // might need to rework this from where it was using pinstransform?
+
+            if (!d[2] || d[2].pinShape !== "none") {
+                var path = d3.select(this).append("path")
+                    .attr("class", "data")
```
Comment (mlzummo) [inline src/map/Pins.js]: @GordonSmith is this really such a good class name for this .... maybe pin-path or pin-shape ?

File: src/map/Pins.js [hunk 2]
```diff
@@ -29,47 +33,143 @@
...
+                    })
+                    .style("stroke", function (d) {
+                        if (d[2] && d[2].strokeColor) {
+                            return d[2].strokeColor;
+                        }
+                        return d[2] && d[2].strokeColor ? d[2].strokeColor : context.pinColor();
+                    })
+                ;
+
+                //var pathBBox = path.node().getBBox();
```
Comment (mlzummo) [inline src/map/Pins.js]: im leavin this for now if thats ok?

**Comment by GordonSmith** [comment]
@mlzummo In the future I am going to auto-close any duplicate PRs like this, its just too painful having switch back and forwards all the time.
I would suggest that It would be far simpler if you just kept committing to the same branch - having lots of commits won't be an issue until we are ready to do the pull (IOW the comment will be "please rollup to single commit to be puller)?

**Comment by mlzummo** [comment]
ok sounds good

**Comment by GordonSmith** [comment]
@mlzummo @jbrundage What is the status of this PR?

**Comment by GordonSmith** [comment]
Please re-submit PR for 1.14.0

**Pull request closed without merging**